### ISSUE TITLE
Feature request: add microphone to the selection.

### ISSUE THREAD

**Issue by AndKe** [Blackstareye/toggleAudio-blackeyeprojects.de#18]
Thank you for the extension.
I find myself toggling between speakers and headset.
Also: when using speakers, the microphone input is the webcam.
When using headset, the microphone is the headset's mic.

It would be very nice if the app had the option to select microphone along with the output device.

**Comment by Blackstareye** [comment]
Thanks for using this extension :) 

I see your usecase there.
But I want to keep the user interface and the function as simple as possible.

So I need to think about that and find a way how I can integrate that into the extension :) 
So it's not off the board but I need a bit of time there so that  this **will be implemented** :) 


PS: I also think about the same mechanism (toogle between inputs) for microphons as another extension

->With that  in mind it would also be possible **to  use macro hotkeys for the hot key kombination of both** :) 

So this could be a workaround as well.

<hr>

**TLDR** I try to add it, but it needs it's time :) 

----

hence you like the extension - little self advertisement here :)

I would appreciate it if you let a review here [review on https://extensions.gnome.org/extension/7080/toggle-headphone/](https://extensions.gnome.org/extension/7080/toggle-headphone/)

This helps the extension to grow so others can see and use this extension as well :)
This also helps to get maybe someday a donation which supports projects like that and helps me doing freelancing stuff :)

**Comment by AndKe** [comment]
Thank you.
I did actually try to leave a review: I emailed info@gnome yesterday to notify them that I was unable to make account due to the verification page being 404:

![image](https://github.com/user-attachments/assets/13274428-06cd-4ce4-9308-d6b0d8457c79)

**Comment by Blackstareye** [comment]
Oh I see. 
I will forward this also to the matrix server :)
Thank you! :)

**Comment by Blackstareye** [comment]
from the matrix channel:

```
Please open an issue for that here:
https://gitlab.gnome.org/Infrastructure/extensions-web
```
@AndKe

**Comment by AndKe** [comment]
done.
Have a great weekend.

**Comment by Blackstareye** [comment]
You already have a good answer in your issue @AndKe  :D  
![grafik](https://github.com/user-attachments/assets/b69b000f-e127-4017-a448-81a046b920a7)

Hope that helps. And Thanks you too! :)
